### PR TITLE
raidboss: ARR Leveling Dungeons Triggers

### PR DESCRIPTION
**Pull request by Aho-Senpai**
Added some triggers for the leveling ARR dungeons.
Translations needed!
(Also, manifest is updated automatically or manually?)
Will add more on separate PRs, waiting to see how this one goes so the next ones can be smoother ^^

### CODE REVIEW

File: ui/raidboss/data/02-arr/dungeon/aurum_vale.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      id: 'Aurum Vale GoldLungs/Burrs',
+      type: 'GainsEffect',
+      // Count `0[2-9]` here is to filter out the first stack
+      netRegex: NetRegexes.gainsEffect({ effectId: ['12E', '12F'], count: '0[2-9]' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Eat Fruit',
+          de: '',
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/dungeon/aurum_vale.ts:22]: In general, leave out lines that don't have translations.  Everything defaults to English if missing and "find missing translations" scripts will not find this if you leave a blank.

File: ui/raidboss/data/02-arr/dungeon/aurum_vale.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheAurumVale,
+  triggers: [
+    {
+      id: 'Aurum Vale GoldLungs/Burrs',
+      type: 'GainsEffect',
+      // Count `0[2-9]` here is to filter out the first stack
+      netRegex: NetRegexes.gainsEffect({ effectId: ['12E', '12F'], count: '0[2-9]' }),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/dungeon/aurum_vale.ts:16]: Isn't it eat after 2 on the first boss, eat after 3 on the final boss, or am I misremembering?

File: ui/raidboss/data/02-arr/dungeon/the_stone_vigil.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheStoneVigil,
+  triggers: [
+    {
+      id: 'Stone Vigil Swinge',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '387', source: 'Chudo-Yudo' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '387' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '387' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '387' }),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/dungeon/the_stone_vigil.ts:18]: Don't add in the other netRegexLang if you don't have the name translated.  Just leave them out and Akurosia will follow up with another PR.

**Review by quisquous — COMMENTED** [review]
Thanks!

> Translations needed!

There's a bunch of scripts that will find missing translations, and so just landing the untranslated PR will get other people to follow up automatically.

> (Also, manifest is updated automatically or manually?)

You need to update it manually.  There is probably a test that will complain if you haven't updated it.

File: ui/raidboss/data/02-arr/dungeon/aurum_vale.ts (hunk 2 above)
Comment (Aho-Senpai) [reply ui/raidboss/data/02-arr/dungeon/aurum_vale.ts:16]: TBH, I'm not 100% sure, but I don't see a real downside to eating it at 2+ on both, nor ever ran into not having enough fruits. Can be changed if you want though

**Comment by Aho-Senpai** [comment]
I'm confused as to why it's failing the NPM test, honestly. 
`Found unnecessary regex capturing group for trigger id 'Stone Vigil Swinge'.`

**Comment by MaikoTan** [comment]
> I'm confused as to why it's failing the NPM test, honestly. `Found unnecessary regex capturing group for trigger id 'Stone Vigil Swinge'.`

RegExp would capture by default, but if you don't need the capture groups, it would be better to just not to capture. For precisely, you should add the property `capture: false` in RegExp factory functions:
```diff
- netRegex: NetRegexes.startsUsing({ id: '387', source: 'Chudo-Yudo' }),
+ netRegex: NetRegexes.startsUsing({ id: '387', source: 'Chudo-Yudo', capture: false }),
```

**Pull request merged**